frodo.js

```diff
@@ -2,48 +2,48 @@
  * Extras Plugin
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.